src/PrestaShopBundle/Form/Admin/Type/DeltaQuantityType.php

```diff
@@ -58,6 +70,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'constraints' => [
                     new Type(['type' => 'numeric']),
                     new NotBlank(),
+                    new Range([
+                        'min' => -static::INT_32_MAX,
+                        'max' => static::INT_32_MAX,
```

Comment:
```suggestion
                        'min' => 2 * -static::INT_32_MAX,
                        'max' => 2 * static::INT_32_MAX,
```

Reply:
Even if there is a bug blocking those high values (see related issue https://github.com/PrestaShop/PrestaShop/issues/29253) we can probably anticipate a little and configure the real expected value. It will still fail for intermediate values but at least the real invalid values are handled this way.